config.ts

```diff
@@ -15,21 +26,29 @@ export let config = {
     : [],
   host: envs.HOST || "localhost",
   port: Number(envs.PORT) || 8000,
-  https: !!envs.HTTPS || false,
-  jwt: !!envs.JWT || false,
+  https: Boolean(envs.HTTPS) || false,
+  jwt: Boolean(envs.JWT) || false,
   refreshTokenDays: Number(envs.REFRESH_TOKEN_DAYS) || 120,
   serverCodeLength: Number(envs.SERVER_CODE_LENGTH) || 256,
   sslCertificateLocation: envs.HTTPS_CERTIFICATE_LOCATION,
   sslKeyLocation: envs.HTTPS_KEY_LOCATION,
-  enableTrafficLogs: envs.ENABLE_TRAFFIC_LOGS || false,
-  version: "0.11.5"
+  enableTrafficLogs: Boolean(envs.ENABLE_TRAFFIC_LOGS) || false,
+  version: "0.11.5",
```

Comment:
```suggestion
  version: "0.12.0",
```
This will be a significant update, thanks to you!